app/gulp/sass/app.scss

```diff
@@ -3,3 +3,25 @@
 @import '_error';
 @import '_layout';
 @import '_misc';
+
+html {
+  height: 100%;
+  width: 100%;
+}
+
+body {
+  height: calc(100% - 60px);
+  width: 100%;
+}
+
+div.tooltip {
+  position: absolute;
+  text-align: center;
+  padding: 2px;
+  font: 12px sans-serif;
+  background: lightgrey;
+  border: 1px;
+  border-color: black;
+  pointer-events: none;
+  opacity: 0.3;
+}
```

Comment:
Is this `div.tooltop` on every page or d3 graph? If not, I would recommend creating a file like `_statistics.scss` and `@import`ing it instead.